### PR TITLE
Add port option.

### PR DESCRIPTION
**Pull request by matt-42**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -60,6 +65,8 @@ runs:
       shell: bash
 
     - name: Setup PostgreSQL user and database
+      env:
+        PGPORT: ${{ inputs.port }}
```
Comment (ikalnytskyi) [inline action.yml:69]: This won't persist, and thus users' steps won't be able to use `psql` or `createuser` directly. We should persist it together with `PGUSER` and `PGHOST` in the step above. Can you please move it there?

**Comment by ikalnytskyi** [comment]
Hey @matt-42,

Thanks for the contribution. Can you please provide some context on why it's needed? I mean, why would someone want to use a non-default PostgreSQL port on their CI? Is there some use case for that?

File: action.yml (hunk 1 above)
Comment (matt-42) [reply action.yml:69]: > 

I'm writing a database driver and I wanted to test that it also work on a non default port. That's why I added this option.

Comment (matt-42) [reply action.yml]: > This won't persist, and thus users' steps won't be able to use `psql` or `createuser` directly. We should persist it together with `PGUSER` and `PGHOST` in the step above. Can you please move it there?

Fixed in my last commit.

Comment (ikalnytskyi) [reply action.yml]: > I'm writing a database driver and I wanted to test that it also work on a non default port. That's why I added this option.

Makes sense. :+1: 

> Fixed in my last commit.

Great! Thank you!

**Review by ikalnytskyi — APPROVED** [review]

**Comment by ikalnytskyi** [comment]
I just published `ikalnytskyi/action-setup-postgres@v2` with this enhancement. Thank you!

**Pull request merged**